features/cellular/TARGET_GENERIC_MODEM/generic_modem_driver/mbed_lib.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "ref-cell-drv", 
+		"config": {
+			"baud-rate": 115200,
+			"apn-lookup": false,
+			"at-parser-buffer-size": 256,
+			"at-parser-timeout": 8000
+	},
+	"target_overrides": {
```

Comment:
This is a nice way to turns things on by default, why not pull out the dependencies explicitly?